statemachine.py

```diff
@@ -24,7 +24,7 @@ def fsm():
     memory = ALProxy("ALMemory", IP, 9559)
 
     asr.setLanguage("English")
-    vocab = ["yes", "no", "good", "great", "sad", "bad", "okay", "quit"]     # can be expanded as needed
+    vocab = ["hi", "hello", "yes", "no", "good", "great", "sad", "bad", "okay", "quit"]     # can be expanded as needed
```

Comment:
```suggestion
    vocab = positive + negative
```
we could simple it to combination of two lists.